src/main/app/controller/courts/AddressController.ts

```diff
@@ -0,0 +1,276 @@
+import autobind from 'autobind-decorator';
+import {AuthedRequest} from '../../../types/AuthedRequest';
+import {Response} from 'express';
+import {AxiosError} from 'axios';
+import {CSRF} from '../../../modules/csrf';
+import {
+  AddressType,
+  AddressValidationResult,
+  CourtAddress,
+  DisplayAddress,
+  DisplayCourtAddresses
+} from '../../../types/CourtAddress';
+import {CourtAddressPageData} from '../../../types/CourtAddressPageData';
+import {SelectItem} from '../../../types/CourtPageData';
+
+@autobind
+export class AddressController {
+
+  getAddressTypesError = 'A problem occurred when retrieving the court address types.';
+  getAddressesError = 'A problem occurred when retrieving the court addresses.';
+  updateAddressError = 'A problem occurred when saving the court addresses.';
+  multipleVisitAddressError = 'Only one visit address is permitted.';
+  typeRequiredError = 'Address Type is required.';
+  addressRequiredError = 'Address is required.';
+  townRequiredError = 'Town is required.';
+  invalidPostcodeError = 'Postcode is invalid.';
+  postcodeMissingError = 'Postcode is required.';
+  postcodeNotFoundError = 'Postcode entered could not be found.';
```

Comment:
Ayisha mentioned on another PR the importance for users to be shown a message indicating that changes are not saved when an update is attempted, such as: `The following postcode could not be found (your changes have not been saved): ';`